### PR TITLE
[New Rule] Script Execution via Automator Workflows

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/762

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/execution_script_via_automator_workflows.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2020/12/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of Automator Workflows process followed by a network connection from it's XPC service. Adversaries may drop a custom workflow template that host
+malicious JXA code as an alternative to using osascript.
```
Comment (threat-punter) [inline rules/macos/execution_script_via_automator_workflows.toml:10]:
```suggestion
Identifies the execution of the Automator Workflows process followed by a network connection from it's XPC service. Adversaries may drop a custom workflow template that hosts
malicious JavaScript for Automation (JXA) code as an alternative to using osascript.
```

File: rules/macos/execution_script_via_automator_workflows.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Workflows Automator Execution"
```
Comment (threat-punter) [inline rules/macos/execution_script_via_automator_workflows.toml:16]:
```suggestion
name = "Suspicious Automator Workflows Execution"
```

Is this the term that's more commonly used?

**Review by threat-punter — APPROVED** [review]

File: rules/macos/execution_script_via_automator_workflows.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2020/12/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the Automator Workflows process followed by a network connection from it's XPC service. Adversaries may drop a custom workflow template that hosts
+malicious JavaScript for Automation (JXA) code as an alternative to using osascript.
```
Comment (bm11100) [inline rules/macos/execution_script_via_automator_workflows.toml:10]: this looks like it needs to be linted

**Review by bm11100 — APPROVED** [review]
just one comment, otherwise LGTM 🎊

**Pull request merged**